jobrunner/lib/docker.py

```diff
@@ -280,6 +280,25 @@ def find_newer_files(volume_name, reference_file):
     return sorted(files)
 
 
+def get_file_size_bytes(volume_name, path):
```

Comment:
The approach of one docker subshell per file is a bit inefficient. I suspect we could do this with a single docker with `find | xargs` subshell that returns all files and their sizes, then pull out the ones we want in python, perhaps?

Reply:
If we're going to extract everything to the high priv workspace anyway (which I think we have to) then it would be easier and more efficient to check the sizes there rather than inside the docker volume.

Reply:
Even better